### ISSUE TITLE
Mixed Content Error

### ISSUE THREAD

**Issue by lexhaynes** [joshwcomeau/Tello#1]
In Chrome Version 61.0.3163.100:

Console error for mixed content.

Mixed Content: The page at 'https://tello.tv/app' was loaded over HTTPS, but requested an insecure image 'http://static.tvmaze.com/uploads/images/original_untouched/120/300071.jpg?fit=crop&crop=entropy&h=96&w=334&auto=enhance&dpr=2'. This request has been blocked; the content must be served over HTTPS.

Same error for following two images:

http://static.tvmaze.com/uploads/images/original_untouched/124/311372.jpg?fit=crop&crop=entropy&h=96&w=334&auto=enhance&dpr=2

http://static.tvmaze.com/uploads/images/original_untouched/129/324431.jpg?fit=crop&crop=entropy&h=96&w=334&auto=enhance&dpr=2

![image](https://user-images.githubusercontent.com/1794298/31736118-d34a8b5c-b411-11e7-80fc-ba416dc3d6e7.png)

**Comment by joshwcomeau** [comment]
Huh, strange! I thought I was serving all TV Maze images over HTTPS.

If you refresh, what happens? On refresh it should serve the images from imgix (I create a copy of the images so I can crop them and serve only the part displayed, rather than a huge portrait-oriented image you can only see a sliver of). The imgix images should definitely be HTTPS!

Thanks for letting me know.

**Comment by lexhaynes** [comment]
Refreshed, no errors now.

**Comment by dannief** [comment]
Still an issue for me. When I add a new show, the episode squares do not show up in the summary nor in the 'Manage Show' section. 

In the screenshot below, Star Trek Discovery doesn't show the episode squares. They will only show up after refreshing the page. Same for all the other shows I have added.

Using Chrome 61

![capture1](https://user-images.githubusercontent.com/6110488/31779872-c7cf24ce-b4b9-11e7-9b34-8d66d17f5760.PNG)
![capture2](https://user-images.githubusercontent.com/6110488/31779875-cc658db6-b4b9-11e7-99c4-68dc42ecb655.PNG)
![image](https://user-images.githubusercontent.com/6110488/31779972-15f0781a-b4ba-11e7-8dab-36abcfcb9947.png)

**Comment by joshwcomeau** [comment]
Yeah, I never did get to the bottom of the HTTPS issue; all images should be over HTTPS, unless maybe certain shows don't have an HTTPS image on TV Maze?

I'll try to find some time to look into this one, but since it's still usable after a refresh, I'll likely prioritize some of the other issues (like it not working at all in Firefox Quantum) first.
